pybossa/cache/projects.py

```diff
@@ -256,7 +256,9 @@ def n_remaining_task_runs(project_id):
                              FROM task_run
                              GROUP BY task_id) AS t
                   ON task.id = t.task_id
-                  WHERE task.project_id=:project_id AND task.state = 'ongoing';''')
+                  WHERE task.project_id=:project_id
+                  AND calibration = 0
```

Comment:
wouldn't number of tasks available for users to contribute to include gold tasks?

Reply:
Yes. but this function calculates "expected task_runs remaining". Gold tasks shouldn't be included since then have infinite redundancy.

Reply:
I didn't get what you mean by gold tasks not to be included because of infinite redundancy. when there are gold tasks available for user to contribute or when no tasks except gold are available, they will be presented to the user. isn't this means remaining tasks available to user to include gold tasks?